api/Window.json

```diff
@@ -357,6 +561,607 @@
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "onuserproximity": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/onuserproximity",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": "15"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "onvrdisplayconnect": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/onvrdisplayconnect",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "API Available on all platforms behind a flag, but currently only works on desktop in an <a href='https://webvr.info/get-chrome/'>experimental version of Chrome</a> (other builds won't return any devices when <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getVRDisplays'><code>Navigator.getVRDisplays()</code></a> is invoked)."
+            },
+            "chrome_android": {
+              "version_added": true,
+              "notes": "Currently only supported by Google Daydream."
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "55",
+              "notes": "Currently only Windows support is enabled by default. Mac support is available in <a href='https://hacks.mozilla.org/2017/06/announcing-webvr-on-mac/'>Firefox Nigthly</a>."
```

Comment:
Hmm, I know almost nothing about WebVR, but at least the associated interfaces apparently work on macOS since version 60: https://developer.mozilla.org/en-US/docs/Web/API/VRDisplay#Browser_compatibility